<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1340</width>
    <height>966</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game Board</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="armyDisplay">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>60</y>
      <width>321</width>
      <height>381</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="armiesTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Armies</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblArmiesTerritories">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Armies from Territories:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblArmiesRegions">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Armies from Regions:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblArmiesCards">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Armies from Cards:</string>
     </property>
    </widget>
    <widget class="QLabel" name="numArmiesTerritories">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>80</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>15</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="numArmiesRegions">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>120</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="numArmiesCards">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>160</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblTotalArmies">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>199</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Total Reinforcements:</string>
     </property>
    </widget>
    <widget class="QLabel" name="numTotalArmies">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>200</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>17</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="cardDisplay">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>460</y>
      <width>321</width>
      <height>311</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QGraphicsView" name="mapView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>961</width>
      <height>761</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbxAddCountries">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>10</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Countries</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="linCountry">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>8</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1340</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
